benchmark/net/chunky_http_client.js

```diff
@@ -0,0 +1,102 @@
+'use strict';
+
+// test HTTP throughput in fragmented header case
+var common = require('../common.js');
+var net = require('net');
+var test = require('../../test/common.js');
+
+var bench = common.createBenchmark(main, {
+  len:  [1, 4, 8, 16, 32, 64, 128],
+  num:  [5, 50, 500, 2000],
+  type: ['send'],
+});
+
+
+function main(conf) {
+  var len = +conf.len;
+  var num = +conf.num;
```

Comment:
Yes, this seems to be the assumption in all of the configuration code in the benchmark client headers net/ section.  I am simply following that pre-existing convention. I did notice that there is a conf.len downstairs line 65 or so. I pushed (rebase -i) a minor cleanup fix for it to use 'len' instead.  But both seem fine and correct to me so I doubt that this is more than a very minor aesthetic clarity issue.